styles.css

```diff
@@ -110,12 +117,314 @@ section {
+  margin-top: 10px;
+  margin-bottom: 20px;
+  width: 70%;
+  color: #172b4d;
+}
+
+.project-year {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.canopy {
+  font-size: 13px;
+  font-weight: bolder;
+  color: #344563;
+}
+
+.role {
+  font-size: 13px;
+  color: #7a869a;
+  font-weight: bolder;
+}
+
+.dot {
+  background-color: #c1c7d0;
+  width: 5px;
+  height: 5px;
+  border-radius: 100%;
+  position: relative;
+  bottom: -7px;
+}
+
+.project-description {
+  line-height: 25px;
+  color: #344563;
+}
+
+.project-technologies {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px 0 25px;
+}
+
+.project-technologies > * {
+  margin-right: 10px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  padding: 5px 10px;
+  color: #6070ff;
+  font-size: 12px;
+}
+
+.project-button {
+  width: 130px;
+  height: 48px;
+  background-color: white;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #396df2;
+  font-weight: 500;
+}
+
+.project-button:hover {
+  background-color: #6070ff;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 500;
+}
+
+.project-button:active {
+  background-color: #2230d2;
+  border: 2px solid #2230d2;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 500;
+}
+
+.project-button:link {
+  background-color: white;
+}
+
+.about {
+  background-color: #fff;
+  margin-top: 114px;
+  border-top-right-radius: 100px;
+  padding: 0 24px;
+  font-family: "Poppins", sans-serif;
+}
+
+.about > * {
+  width: 100%;
+}
+
+.about-intro-section p:first-child {
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: bolder;
+  margin-bottom: 0;
+}
+
+.connect-about {
+  color: #7f8cff;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.about-intro-section p:nth-child(2) {
+  color: #344563;
+  padding-top: 0;
+  margin-top: 0;
+}
+
+.social-container-about {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  margin: 26px 0 20px;
+  padding: 0 0 0 12px;
+}
+
+.social-container-about li {
+  margin-right: 20px;
+  font-size: 25px;
   width: 20px;
+  color: #505f79;
+}
+
+.about .project-button {
+  width: auto;
+  padding-right: 12px;
+  padding-left: 12px;
+  font-size: 17px;
+}
+
+.tech-stacks {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.lang-and-icon {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin-top: 50px;
+  margin-bottom: 36px;
+}
+
+.lang-and-icon span:first-child {
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.fa-chevron-down,
+.fa-chevron-right {
+  color: #505f79;
+}
+
+.langs-frames-skills {
+  margin: 0 15px;
+  width: 90%;
+}
+
+.lang-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.lang {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 0 0 10.72px;
+  padding: 10px;
+  background-color: #f7f7f9;
+}
```

Comment:
- [ ] DRY stands for "Don't Repeat Yourself." The main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. You are repeating display:flex multiple times please fix it. for more details check [this](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiUm4qVsNbxAhXK8qQKHWvLA04QFjAFegQIJRAD&url=https%3A%2F%2Fvanseodesign.com%2Fcss%2Fdry-principles%2F&usg=AOvVaw1D-lgJ8nLmTJ-wx73UHUcP)

Reply:
What actually happened here is a case of nested flexbox as we were taught. I have an element that is a flexbox, but one of its flex items still has children that I needed to make flex and make them stack in a row.

I really appreciate the resource you attached to this comment. It really helped me too.